### ISSUE TITLE
khard: passthrough.tests are broken

### ISSUE THREAD

**Issue by antonmosich** [NixOS/nixpkgs#252813]
### Describe the bug
When trying to run the khard tests (by running something like `nix-build -A khard.tests .`), it fails with an error related to ruamel.yaml.
Log:
```console
khard-0.18.0-test-version> building '/nix/store/3qdzdzwbgly64b25wmcgxcll3259ijf8-khard-0.18.0-test-version.drv'
khard-0.18.0-test-version> khard --version returned a non-zero exit code.
khard-0.18.0-test-version> Traceback (most recent call last):
khard-0.18.0-test-version>   File "/nix/store/pb2b805g3z20rqapna029xim3swcq63w-khard-0.18.0/bin/.khard-wrapped", line 6, in <module>
khard-0.18.0-test-version>     from khard.khard import main
khard-0.18.0-test-version>   File "/nix/store/pb2b805g3z20rqapna029xim3swcq63w-khard-0.18.0/lib/python3.10/site-packages/khard/khard.py", line 17, in <module>
khard-0.18.0-test-version>     from . import helpers
khard-0.18.0-test-version>   File "/nix/store/pb2b805g3z20rqapna029xim3swcq63w-khard-0.18.0/lib/python3.10/site-packages/khard/helpers/__init__.py", line 9, in <module>
khard-0.18.0-test-version>     from ruamel.yaml.scalarstring import LiteralScalarString
khard-0.18.0-test-version> ModuleNotFoundError: No module named 'ruamel.yaml'
```

### Steps To Reproduce
1. Just build khard.tests

### Expected behavior
Tests passing without error

### Additional context
I ran git bisect, and the bug seems to be introduced in 5330b05d9952aad7606d5c0658d65faf1955c659, a commit which updated ruamel.yaml. I also ran hg bisect on that repo, and the tests broke in revision `836:18eb08865c03` of the hg repo, which is a rather big changeset.

### Notify maintainers
@matthiasbeyer 

### Metadata

```console
 - system: `"x86_64-linux"`
 - host os: `Linux 6.4.12-1-MANJARO, Manjaro Linux, noversion, rolling`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.15.1`
 - channels(root): `"nixpkgs"`
 - channels(anton): `"home-manager"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixpkgs`

```

**Comment by matthiasbeyer** [comment]
> nix-build -A khard.tests

cannot reproduce on latest master

**Comment by antonmosich** [comment]
It was apparently already fixed in 2b2ff2019e44d8aabf8985dce89955a2f8678312. Thanks for checking though.
